plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -49,6 +49,64 @@ jQuery( function ( $ ) {
 					'.wc_input_variations_price',
 					this.maybe_enable_button_to_add_price_to_variations
 				);
+
+			$( '.save-variations' ).on( 'click', function () {
+
+				$( '#variable_product_options' ).block( {
```

Comment:
What happens when the attribute name and/or values are empty? I wonder if it would be worth checking that before continuing with the process.

Reply:
That's an excellent catch. It would break everything :(